tools/env/base.ts

```diff
@@ -0,0 +1,5 @@
+export = {
+  // Sample API url
+  API: "https://demo.com"
```

Comment:
`$ npm run serve.prod` fails because tslint fails:

```bash
[16:53:31] Error in plugin 'run-sequence(tslint)'
Message:
    Failed to lint: env/base.ts [3, 8]: " should be '.
```

After a change to `API: 'https://demo.com'` it runs through fine.

However the prod env values do not seem to be properly injected:

![screenshot 2016-08-04 17 00 07](https://cloud.githubusercontent.com/assets/1188033/17406743/0cd9e08c-5a65-11e6-9986-e28b79fe5676.png)

Reproduction steps:

1. `$ npm run serve.prod` (`$ npm run build.prod && http-server dist/prod` works too).
2. Open DevTools in Browser and inspect Console.
3. Check environment config:

```
Environment config Object {API: "https://demo.com", ENV: "DEV"}
  API: "https://demo.com"
  ENV: "PROD"
  __proto__: Object
```

Actual Values:

```
Environment config Object {API: "https://demo.com", ENV: "DEV"}
  API: "https://demo.com"
  ENV: "DEV"
  __proto__: Object
```

I'll add a commit to fix the quotes.